cmake/CeleritasLibrary.cmake

```diff
@@ -162,9 +159,14 @@ endfunction()
 #
 function(celeritas_add_library target)
 
-  set(_midsuf "")
-  set(_staticsuf "_static")
```

Comment:
OK, I thought that since `_final` was hardcoded I could just hardcode these as well. I think since it's localized to this function and we're not customizing them, it improves readability a bit to just inline them?

Reply:
I suspect I could/should have treated `_final` the same way.   Removing the place holder for customization makes it much harder to add later.  The code in this function is intended to be more generic than just celeritas and diverging too much means from the "going to" be upstream (VecGeom and/or https://github.com/drbenmorgan/CudaRDCExercise) would make it harder to carry through (up and down) bug fix if any.

Further, I actually thing that the code is more readable with the clear marker/reminder of the intended of each of the libraries.  (In that routine `${target}${_midsuf}` and `${target}` are not semantically the same per se).

Reply:
@pcanal I'm looking at this again, and it seems to me that "midsuf" should *always* be empty because that's the library that contains all the object code for the target name you're asking to create. The "staticsuf" changes based on whether it's static or not. The other two suffixes, `_final` and `_objects`, were hardcoded before my changes. Under what circumstances would `midsuf` be  different? Maybe if we switched `_final` and `${_midsuf}`? So can we just leave it hardcoded for midsuf as well until someone requests the extra flexibility?

Reply:
fair enough